database/seeds/StripeSupportedCurrencySeeder.php

```diff
@@ -14,807 +14,807 @@ class StripeSupportedCurrencySeeder extends Seeder
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MUR',
+         StripePayment::create([
+		'currencies'             => 'MUR',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MRO',
+         StripePayment::create([
+		'currencies'             => 'MRO',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MOP',
+         StripePayment::create([
+		'currencies'             => 'MOP',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MNT',
+         StripePayment::create([
+		'currencies'             => 'MNT',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MMK',
+         StripePayment::create([
+		'currencies'             => 'MMK',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MKD',
+         StripePayment::create([
+		'currencies'             => 'MKD',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MGA',
+         StripePayment::create([
+		'currencies'             => 'MGA',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-         Stripe::create([
-		'supported_currencies'             => 'MDL',
+         StripePayment::create([
+		'currencies'             => 'MDL',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-          Stripe::create([
-		'supported_currencies'             => 'MAD',
+          StripePayment::create([
+		'currencies'             => 'MAD',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-           Stripe::create([
-		'supported_currencies'             => 'LSL',
+           StripePayment::create([
+		'currencies'             => 'LSL',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-            Stripe::create([
-		'supported_currencies'             => 'LRD',
+            StripePayment::create([
+		'currencies'             => 'LRD',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
 
-             Stripe::create([
-		'supported_currencies'             => 'LKR',
+             StripePayment::create([
+		'currencies'             => 'LKR',
 		'base_currency'               => 'USD',
 		'processing_fee'               => '5',
 		]);
```

Comment:
Are we going to truncate the table and then run this seeder?

Reply:
No, we create the table and soon after seed these values